gamemodes/terrortown/gamemode/player_ext_shd.lua

```diff
@@ -563,6 +563,19 @@ if CLIENT then
         end
         emitter:Finish()
     end
+
+    function plymeta:QueueMessage(type, message, time)
+        if LocalPlayer() ~= self then
+            ErrorNoHalt("`plymeta:QueueMessage` cannot be used to send messages to other players when called clientside.")
```

Comment:
`ErrorNoHalt` doesn't end with a newline automatically
```suggestion
            ErrorNoHalt("`plymeta:QueueMessage` cannot be used to send messages to other players when called clientside.\n")
```